Error when submitting large import spec

## Expected Behavior 
Import job should be submitted successfully

## Current Behavior
Submitting an import job with large import spec causes following exception

```
Caused by: org.postgresql.util.PSQLException: ERROR: value too long for type character varying(2048)
	at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2440)
	at org.postgresql.core.v3.QueryExecutorImpl.processResults(QueryExecutorImpl.java:2183)
	at org.postgresql.core.v3.QueryExecutorImpl.execute(QueryExecutorImpl.java:308)
	at org.postgresql.jdbc.PgStatement.executeInternal(PgStatement.java:441)
	at org.postgresql.jdbc.PgStatement.execute(PgStatement.java:365)
	at org.postgresql.jdbc.PgPreparedStatement.executeWithFlags(PgPreparedStatement.java:143)
	at org.postgresql.jdbc.PgPreparedStatement.executeUpdate(PgPreparedStatement.java:120)
	at com.zaxxer.hikari.pool.ProxyPreparedStatement.executeUpdate(ProxyPreparedStatement.java:61)
	at com.zaxxer.hikari.pool.HikariProxyPreparedStatement.executeUpdate(HikariProxyPreparedStatement.java)
	at org.hibernate.engine.jdbc.internal.ResultSetReturnImpl.executeUpdate(ResultSetReturnImpl.java:175)
```

--- a/core/src/main/java/feast/core/model/JobInfo.java
+++ b/core/src/main/java/feast/core/model/JobInfo.java
@@ -81,7 +81,7 @@ public class JobInfo extends AbstractTimestampEntity {
   private JobStatus status;
 
   // Raw import spec, stored as a json string.
-  @Column(name = "raw", length = 2048)
+  @Column(name = "raw", length = 4096)
   private String raw;
 
   public JobInfo() {

